airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -101,7 +101,11 @@ def execute(self, context):
         if self.script_location and not self.script_location.startswith(self.s3_protocol):
             s3_hook = S3Hook(aws_conn_id=self.aws_conn_id)
             script_name = os.path.basename(self.script_location)
-            s3_hook.load_file(self.script_location, self.s3_bucket, self.s3_artifacts_prefix + script_name)
+            s3_hook.load_file(
+                filename=self.script_location,
+                key=self.s3_artifacts_prefix + script_name,
+                bucket_name=self.s3_bucket,
```

Comment:
Thanks for fixing this. would it be possible to also add an argument to replace if the file exists, which is the `replace` operator?

Reply:
I think I will merge it as is as we are releasing providers tomorrow/Monday. This might be a separate feature.